CHANGELOG.md

```diff
@@ -3,3 +3,4 @@
 2015-12-17 - v0.2.0 Better validation when changing relations
 2015-12-17 - v0.3.0 Fixes to enable usage in web browsers
 2015-12-17 - v0.4.0 Better handling of non-json:api payloads
+2016-02-08 - v0.4.1 Handling resources with no relationships
```

Comment:
Can we reorder this to be most recent one first? Also add `-` or `*` before the entries so it is valid markdown and renders correctly in github (if you press the `View` button you'll see what I mean).